mod_test/models.py

```diff
@@ -328,10 +329,11 @@ def __repr__(self):
         Represent a TestResult Model by its 'test_id','expected_rc',
          'regression_test_id' and 'status' Field.
 
-        :return id,exit_code,regression_test_id,expected_rc,runtime:
-            Returns the 'id' , 'exit_code', 'regression_test_id',
-             'expected_rc', 'runtime' field of the TestResult model
-        :rtype id,exit_code,regression_test_id,expected_rc,runtime: str
+        :return obj(id,exit_code,regression_test_id,expected_rc,runtime):
```

Comment:
You're not returning an object... You return a string representation which is based on the properties of the object